var/spack/repos/builtin/packages/py-ipython/package.py

```diff
@@ -44,7 +44,7 @@ class PyIpython(PythonPackage):
     depends_on('py-prompt-toolkit@2.0.0:2.0.999', when='@7.5.0', type=('build', 'run'))
     depends_on('py-prompt-toolkit@2.0.0:2.999,3.0.2:3.0.999', when='@7.18:', type=('build', 'run'))
     depends_on('py-pygments', type=('build', 'run'))
-    depends_on('py-backcall', type=('build', 'run'), when='@7.5.0:')
+    depends_on('py-backcall', type=('build', 'run'), when='@7.3.0:')
```

Comment:
This looks unrelated, accidentally fell into this branch?

Reply:
Yes, the ipython is accidental, I must have made an update for that in a wrong branch. Sorry about that and thanks for accepting the Octave pull request. Do I need to undo the ipython change by updating this pull request or can you do it?

Reply:
I don't think I have access to push to your branch, it would be easier if you could make the change, either with a rebase or a new commit, I don't think it matters here.